src/components/JapaneseText.tsx

```
@@ -0,0 +1,13 @@
+import React from 'react';
+
+interface JapaneseTextProps {
+  text: string;
+}
+
+const JapaneseText: React.FC<JapaneseTextProps> = ({ text }) => {
+  return (
+    <div dangerouslySetInnerHTML={{ __html: text.replace(/\*\*([^*]+)\*\*/g, '<b>$1</b>') }} />
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)